addons/website/static/src/snippets/s_timeline/001.scss

```
@@ -0,0 +1,25 @@
+.s_timeline[data-vcss="001"] {
+    .s_timeline_dot {
+        @include media-breakpoint-up(md) {
+            left: 50%;
+        }
+
+        padding: var(--o-timeline-size, 12px);
+        outline: map-get($spacers, 1) solid var(--o-cc-bg, #{$body-bg});
```

Nitpicking, regular rules before bedia queries
```suggestion
        padding: var(--o-timeline-size, 12px);
        outline: map-get($spacers, 1) solid var(--o-cc-bg, #{$body-bg});
        
        @include media-breakpoint-up(md) {
            left: 50%;
        }
```